### DISCUSSION TITLE
Is KEDA required if Kubernetes executor is used?

### DISCUSSION THREAD

**Discussion by y0zg** [apache/airflow#30525] [Q&A]
Considering airflow is deployed in kubernetes and Kubernetes executor is used, do you enable autoscaling? If yes, do you use KEDA? Based on which metrics do you scale?

**Comment by potiuk** [comment]
Depends on the chart you use. It's not "airflow" it's "deployment" part so it depends how Airflow is deployed (note Airflow does NOT have to be run in k8s).

This is our chart's KEDA information https://airflow.apache.org/docs/helm-chart/stable/keda.html (and you can find out more by looking at the source code if you want more details).

How it is in other charts - you need ot check yourself.

BTW. I saw you copy & pasted the same question in slack to- suggest to search the docs first. They often contain information you need.
